src/App.js

```diff
@@ -1,20 +1,22 @@
 import './App.css';
 
 function App() {
+  const x = [true, true, false]
   return (
     <div class="container">
       <div class="todo">My Todo App</div>
-      <div class="workspace">
-        <div class="list">
-          <img src="https://img.icons8.com/ios/30/1ABC9C/unchecked-circle.png" class="image" /> Buy the groceries
-        </div>
-        <div class="list">
-          <img src="https://img.icons8.com/ios/30/1ABC9C/unchecked-circle.png" class="image" /> Get the items
-        </div>
-        <div class="list">
-          <img src="https://img.icons8.com/ios/30/1ABC9C/unchecked-circle.png" class="image" /> Push to Github
-        </div>
-      </div>
+      <div class="workspace"></div>
+      {x.map(item => {
+        if (item == true) {
+          return (<div class="list">
+            <img src="https://img.icons8.com/ios/30/1ABC9C/unchecked-circle.png" class="image" /> Buy the groceries
```

Comment:
enclose the "Buy the groceries" inside a div to make it consistent with the checked item